package.json

```diff
@@ -6,10 +6,10 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
```

Comment:
We shouldn't replace an existing script, as we might still need it. Maybe give your script a different name?

Reply:
I thought about it, but ideally there should be 1 command for running all tests, which is jest is great for, considering that in the future, react tests could be written using jest. 

For now, I can bring that command back and rename it to something along the way of `pwa:test`. Let me know what you think

Reply:
Fair enough.